Tools/GenerateFullWebsiteData.ps1

```diff
@@ -0,0 +1,82 @@
+# Generates a JSON file containing the documentation information for all
```

Comment:
This file is just a copy of `GenerateWebsiteData.ps1` right? Could we just have that script generate two sets of files? One for the old website and another set of JSON files for your new one? Otherwise we'd need to make updates in two places to this script.